test/compilable/fieldwise.d

```diff
@@ -0,0 +1,11 @@
+/* REQUIRED_ARGS: -preview=dip1000 -preview=fieldwise
+ */
+
+import imports.impfieldwise;
+
+@safe:
+
+bool test(S s, S t)
+{
+    return s == t; // comparison can access fields for ==
+}
```

Comment:
Do we have a test that verifies that it's not possible to compared the fields directory?

Reply:
Yes, I found this problem because it was failing existing tests.